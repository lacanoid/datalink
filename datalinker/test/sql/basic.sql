\pset null _null_

select *
  from datalink.dl_space('/etc');
  
select *
  from datalink.dl_space('/tmp/');
  
select *
  from datalink.dl_space('/tmp/zzzz');

-----------------------------------------

select *
  from datalink.dl_paths('/etc');
  
select *
  from datalink.dl_paths('/tmp/');
  
select *
  from datalink.dl_paths('/tmp/zzzz');

-----------------------------------------

select *
  from datalink.dl_inode('/etc');

select *
  from datalink.dl_inode('/tmp/');
  
select *
  from datalink.dl_inode('/tmp/zzzz');
  
  