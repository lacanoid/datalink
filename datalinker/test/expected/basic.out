\pset null _null_
select *
  from datalink.dl_space('/etc');
 space_id | base_path | prefix | archive_path 
----------+-----------+--------+--------------
   _null_ | _null_    | _null_ | _null_
(1 row)

  
select *
  from datalink.dl_space('/tmp/');
 space_id | base_path | prefix | archive_path 
----------+-----------+--------+--------------
        1 | /tmp      | _null_ | _null_
(1 row)

  
select *
  from datalink.dl_space('/tmp/zzzz');
 space_id | base_path | prefix | archive_path 
----------+-----------+--------+--------------
        1 | /tmp      | _null_ | _null_
(1 row)

  
  
